go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckhouse/deckhouse/dhctl v0.0.0 // use non-existent version for replace
 	github.com/fatih/color v1.13.0
-	github.com/flant/addon-operator v1.3.13-0.20240527095525-c2870b9f03b0
+	github.com/flant/addon-operator v1.3.13-0.20240627133316-cfb2b3d0ca9b
```

Comment:
It will be necessary to update the `addon-operator` version after we create a new version in it.